analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -645,5 +645,5 @@ open class Npm(
     }
 
     protected open fun runInstall(workingDir: File) =
-        run(workingDir, if (hasLockFile(workingDir)) "ci" else "install", "--ignore-scripts")
+        run(workingDir, if (hasLockFile(workingDir)) "ci" else "install", "--ignore-scripts --no-audit")
```

Comment:
`"--no-audit"` should be passed as a separate argument to `run`.

Reply:
And just checking, is there a similar logic in Yarn that we should disable, too?

Reply:
For yarn:

>  Unlike npm, which automatically runs an audit on every install, yarn will only do so when requested.

https://classic.yarnpkg.com/en/docs/cli/install

So nothing is needed.